website/docs/cdktf/release/upgrade-guide-v0-9.mdx

```diff
@@ -0,0 +1,38 @@
+---
+page_title: Upgrading to CDKTF Version 0.9
+description: >-
+  We changed some provider bindings to work better across all languages.
+---
+
+# Upgrading to CDK for Terraform Version 0.9
+
+We changed some provider bindings to work better across all languages.
+
+### Standardize IResolvable Usage [#1299](https://github.com/hashicorp/terraform-cdk/pull/1299)
+
+This is an effort to make sure attributes can be freely passed between resources for all different types.
+
+There is a minor breaking change:
+
+- `count` on resources and data sources has gone from `number | cdktf.IResolvable` to `number`. If code was previously passing an `IResolvable`, it will now need to use `Token.asNumber()`
```

Comment:
I think it could be helpful to add examples for other languages.
For example Go uses `Token_AsNumber` I think 😄

Reply:
You can have a look at the ucloud examples I changed here: https://github.com/hashicorp/terraform-cdk/pull/1499/files#diff-475c889d36864541f858ca1bca543d2f82156137ebdbed3a7e24a88def646e7bR28
They should cover all the languages for the Token.asXy changes

Reply:
Yeah good pointed, pushed a fix :)